src/game/client/baseheightmap.cpp

```diff
@@ -966,91 +966,96 @@ void BaseHeightMapRenderObjClass::Do_The_Light(VertexFormatXYZDUV2 *vb,
         alpha);
 }
 
-float BaseHeightMapRenderObjClass::Get_Height_Map_Height(float x, float y, Coord3D *pos) const
+float BaseHeightMapRenderObjClass::Get_Height_Map_Height(float input_x, float input_y, Coord3D *pos) const

```

Comment:
Perhaps can leave x, y as is. It is short and also in the declaration of the function. Generally if function definition argument names are changed, then those of function declaration should be changed too.

Reply:
Done. Also formatted.